rewriter/Flatten.cc

```diff
@@ -341,7 +341,7 @@ class FlattenWalk {
         // if we get a MethodData back, then we need to move this and replace it with an EmptyTree
         if (auto md = methods.popScope()) {
             methods.addExpr(*md, move(tree));
-            tree = ast::MK::EmptyTree();
+            tree = ast::MK::Nil(send.loc);
```

Comment:
Is the rationale here that we need to replace it with something the user could have actually written, so that we don't see `private(<empty-tree>)`?

Probably want to update the comment right before the `if`, maybe also adding an explanation of why we can't use `EmptyTree` here.